dashboard/views/index.ejs

```diff
@@ -20,7 +20,10 @@
 </div>
 
 <script data-api-base-url="<%= apiBaseUrl %>" data-websocket="<%= websocket %>" src="./main.js"></script>
+<script>
+  const config = JSON.parse('<%- uiConfig %>');
```

Comment:
Could that be written like that?

`const config = <%- uiConfig %>;`

Reply:
No as uiConfig needs to be stringified before sent here as we need strings here, not JS.